### PR TITLE
Add README.md description

### PR DESCRIPTION
**Pull request by FedenkoMykhailo**
Any ideas regarding the content and design of the README.md file are accepted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+### 3.1. Required to install
+
+* Java 17
+* PostgreSQL
+
+### 3.2. How to run
+
+1. You should create database `pharmagator` and set your db username and password in application.yml .
+
+## Special Thanks :heart:
```
Comment (Oleksandr-Eleks) [inline README.md:26]: Sounds great! =)))
But please remove this section, I'm a bit shy! =)))))

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+## 3. Start the project locally
+
+### 3.1. Required to install
+
+* Java 17
+* PostgreSQL
+
+### 3.2. How to run
+
+1. You should create database `pharmagator` and set your db username and password in application.yml .
```
Comment (Oleksandr-Eleks) [inline README.md:24]: To avoid misconfigurations username and pwd should be set as 'postgres'

File: README.md (hunk 1 above)
Comment (FedenkoMykhailo) [reply README.md]: Removed this section

File: README.md (hunk 2 above)
Comment (FedenkoMykhailo) [reply README.md]: Specified config details

**Review by Oleksandr-Eleks — APPROVED** [review]

**Review by MariamnaKh — DISMISSED** [review]

**Review by MariamnaKh — APPROVED** [review]

**Review by Oleksandr-Eleks — APPROVED** [review]

**Review by MariamnaKh — APPROVED** [review]

**Pull request merged**